src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -662,6 +664,80 @@ static String getValidityInfoForSizeCappedPossiblyEmptyString(String fieldName,
         return "";
     }
 
+    /**
+     * Checks if Session Start Time is after 3 hours before now, before 90 days from now and at exact hour mark.
+     * @return Error string if {@code sessionStart} is more than 3 hours before now, after 90 days from now
+     *         or not at exact hour mark
+     *         Empty string if {@code sessionStart} is after 3 hours before now, before 90 days from now
+     *         and at exact hour mark
```

Comment:
This is too detailed. Javadoc header comment, at least the first paragraph, is just for method overview. In other words this is sufficient:
```java
/**
 * Checks if the given Instant is valid to be used as a session start time.
 * Returns an empty string if it is valid, or an error message otherwise.
```
The implementation details, you can either put it within the method body itself, or in a separate paragraph in the header comment.